### ISSUE TITLE
use relative time

### ISSUE THREAD

**Issue by MAHcodes** [nodetec/notebin#144]

**Comment by MAHcodes** [comment]
https://www.youtube.com/watch?v=acemrBKuDqw

**Comment by ChristianChiarulli** [comment]
what should we use relative time for?

**Comment by MAHcodes** [comment]
note date
![image](https://user-images.githubusercontent.com/69465962/214705762-cf0daf94-a2bb-478c-8dfd-0fc92fe0ba4a.png)

**Comment by ChristianChiarulli** [comment]
is this for users in different timezones?

**Comment by MAHcodes** [comment]
no, it'll show "yesterday" instead of "Jan 25, 2023", or "today", "2 hours ago", and so on... here's the library for it, https://momentjs.com/

**Comment by ChristianChiarulli** [comment]
ah, yea this would be nice
